styles.css

```diff
@@ -33,6 +33,7 @@ h3 {
 main {
   display: flex;
   padding: 10px 20px 20px 20px;
+  align-items:flex-start;
```

Comment:
This keeps the columns content at the top of the containers when another section becomes longer.